### PR TITLE
[Docs] Update MLRun CE Kubernetes Installation Docs for new "only full" Deployment

### PR DESCRIPTION
**Pull request by quaark**
- Some improvements to make the install docs clearer.
- Now there is only a full installation with options to opt out of some features

### CODE REVIEW

File: docs/install/kubernetes.md [hunk 1]
```diff
@@ -96,6 +96,16 @@ Add the Community Edition helm chart repo:
 helm repo add mlrun-ce https://mlrun.github.io/ce
 ```
 
+Run the following command to ensure the repo is installed and available:
+```bash
+helm repo list
+```
+It should outuput something like:
```
Comment (jillnogold) [inline docs/install/kubernetes.md:103]: It should output something like:

File: docs/install/kubernetes.md [hunk 2]
```diff
@@ -96,6 +96,16 @@ Add the Community Edition helm chart repo:
 helm repo add mlrun-ce https://mlrun.github.io/ce
 ```
 
+Run the following command to ensure the repo is installed and available:
```
Comment (jillnogold) [inline docs/install/kubernetes.md:99]: Run the following command to ensure that the repo is installed and available:

File: docs/install/kubernetes.md [hunk 3]
```diff
@@ -147,14 +157,15 @@ When the installation is complete, the helm command prints the URLs and Ports of
 
 ## Installing the Full Version
 
-To install the full version of the chart, use the following command:
+To install the full version of the chart first follow the instructions of installing the light version up until the helm
```
Comment (jillnogold) [inline docs/install/kubernetes.md:160]: To install the full version of the chart, follow the instructions of installing the light version up to an including creating a secret with your docker-registry, and then continue with:

**Review by Tankilevitch — APPROVED** [review]

File: docs/install/kubernetes.md [hunk 4]
```diff
@@ -214,11 +215,26 @@ In order to opt out of some of the components, you can use the following helm va
...
+## Storage resources
+
+When installing the MLRun Community Edition, several storage resources are created:
+
+- **PVs via default configured storage class**: Used to hold the file system of the stacks pods, including the MySQL database of MLRun, Minio for artifacts and Pipelines Storage and more. These are deleted when the stack is uninstalled.
+- **Container Images in the configured docker-registry**: When building and deploying MLRun and Nuclio functions via the MLRun Community Edition, the function images are stored in the given configured docker registry. These images persist in the docker registry and are not deleted.
+
 ## Uninstalling the Chart
+
+The following command will delete the pods, deployments, config maps, services, roles+role bindings and PVCs associated with the chart and release.
```
Comment (jillnogold) [inline docs/install/kubernetes.md:231]: The following command deletes the pods, deployments, config maps, services, roles+role bindings and PVCs associated with the chart and release.

File: docs/install/kubernetes.md [hunk 5]
```diff
@@ -5,12 +5,12 @@
 - [Prerequisites](#prerequisites)
 - [Community Edition Flavors](#community-edition-flavors)
 - [Installing on Docker Desktop](#installing-on-docker-desktop)
-- [Installing the Lite Version](#installing-the-lite-version)
-- [Installing the Full Version](#installing-the-full-version)
+- [Installing the Chart](#installing-the-chart)
 - [Configuring Online Feature Store](#configuring-online-feature-store)
 - [Start working](#start-working)
```
Comment (jillnogold) [inline docs/install/kubernetes.md:10]: Add after Configuring Online Feature Store
- [Usage](#usage)

**Pull request merged**